### DISCUSSION TITLE
How to listen to a large range of ports or any port on localhost?

### DISCUSSION THREAD

**Discussion by AgentRG** [mitmproxy/mitmproxy#6044] [Q&A]
I see that by default, the server listen to traffic going to 8080, and it can also be specified to a specific port. I have a situation where a specific traffic I am trying to capture and modify the response of can be triggered on any port in the range of 50000-60000.

Does mitmproxy have a way to listen to any port within that range? The URL can look something the lines of `http://127.0.0.1:55684/the-rest-of-the-url`

**Comment by sujaldev** [comment]
If you're on linux, you could achieve this with iptables. Check [this](https://superuser.com/questions/440324/iptables-how-to-forward-all-external-ports-to-one-local-port) post.

**Reply by AgentRG** [reply]
Regretfully, hard stuck on a macOS with this. But thanks for the reply.

**Reply by sujaldev** [reply]
There's an [equivlanet](https://serverfault.com/questions/102416/iptables-equivalent-for-mac-os-x) for macOS.

**Comment by ericbeland** [comment]
Another option is that you can run linux in docker or on a VM and use IPtables.
